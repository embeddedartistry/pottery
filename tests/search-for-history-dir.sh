pottery init
pottery post First Record
mkdir subdir
cd subdir
pottery post Second Record
ls docs/project-history
cd ..
find docs/project-history -print | sort

