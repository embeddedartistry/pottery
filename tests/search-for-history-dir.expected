pottery init
pottery post First Record
mkdir subdir
cd subdir
pottery post Second Record
ls docs/project-history
ls: docs/project-history: No such file or directory
cd ..
find docs/project-history -print | sort
docs/project-history
docs/project-history/1992
docs/project-history/1992/1992-01
docs/project-history/1992/1992-01/1992-01-12T16:34:23Z.md

